pkl-doc/src/test/kotlin/org/pkl/doc/CliDocGeneratorTest.kt

```diff
@@ -219,6 +226,11 @@ class CliDocGeneratorTest {
       .withFailMessage("Test bug: $actualFile should exist but does not.")
       .exists()
 
+    // symlinks on Git and Windows is rather finnicky; they create shortcuts by default unless
+    // a core Git option is set. Also, by default, symlinks require administrator privileges to run.
```

Comment:
... but a `.lnk` is a text file with the redirected path in, no? Do you also need admin to read? If not; Is it an option to read `actualFile` instead and replace it with the contents? (Would possibly require a `while`, because of link-to-link cases.)

Reply:
As far as I can tell, there's no easy way to parse a `lnk` file in Java and figure out where it's pointed to.